pkg/tests/api/correlations/correlations_update_test.go

```diff
@@ -6,8 +6,10 @@ import (
 	"io"
 	"net/http"
 	"testing"
+	"time"
 
 	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/assert"
```

Comment:
should not need assert, require is an alternate that fails fast. So should be able to replace assert. below with require I think (or if it is needed to `.CollecT`, isn't needed for the `.Equal`